spec/stack.js

```diff
@@ -19,4 +19,24 @@ describe('Stack', () => {
         .to.alter(() => myStack.size(), { from: 0, to: 1 })
     })
   })
+
+
+  context('remove()', () => {
+    it('removes an element from the top of the stack.', () => {
+      const myStack = new Stack()
+      myStack.add('foo')
```

Comment:
consider using mocha's `beforeEach()` hook to avoid repeated code. https://mochajs.org/#hooks